src/deepsparse/loggers/metric_functions/cv/__init__.py

```diff
@@ -0,0 +1,17 @@
+# flake8: noqa
```

Comment:
let's rename this directory to `computer_vision`. we generally standardize to spelling things out